core/variant/variant_parser.cpp

```diff
@@ -1123,6 +1124,16 @@ Error VariantParser::parse_value(Token &token, Variant &value, Stream *p_stream,
 				get_token(p_stream, token, line, r_err_str);
 				if (token.type == TK_STRING) {
 					String path = token.value;
+					if (p_res_parser != nullptr) {
+						String *uid = p_res_parser->path_to_uid.getptr(path);
```

Comment:
Since we also have `ResourceUID *resource_uid` I think this needs clarification.

```suggestion
						String *uid_string = p_res_parser->path_to_uid.getptr(path);
```